uaclient/tests/test_cli_collect_logs.py

```diff
@@ -51,6 +51,10 @@ def test_collect_logs_help(self, _m_resources, _getuid, capsys):
         out, _err = capsys.readouterr()
         assert re.match(HELP_OUTPUT, out)
 
+    @mock.patch(
+        M_PATH + "glob.glob",
+        return_value=["ubuntu-advantage.log", "ubuntu-advantage.log.1"],
+    )
```

Comment:
Don't know if this is feasible,  but can't we mock `os.path.isfile` just to check which files are being read by the command ?

Reply:
Yeah I can do something like that, good idea!